src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
+    pub bytes_per_line: usize,
```

Comment:
Controls how many bytes per line will be displayed. 

Question for you: What does happen on 0?